nifi-commons/nifi-data-provenance-utils/src/test/groovy/org/apache/nifi/provenance/CryptoUtilsTest.groovy

```diff
@@ -88,6 +88,11 @@ class CryptoUtilsTest {
         Cipher.getMaxAllowedKeyLength("AES") > 128
     }
 
+    private static boolean isRootUser() {
+        Process p = Runtime.getRuntime().exec("id -u")
+        p.exitValue() == 0
```

Comment:
Yep, excellent catch. I'll fix it to evaluate the response, not the exit code. Thanks.